packages/docs/README.md

```diff
@@ -0,0 +1,76 @@
+#TODO
+@async
+@author
+@ignore
+@see
+@summary
+
+# docs
+
+In order for comments to be processed by the docs system they must begin with \*
+
+Below these types of comments are referred to as 'starred' or 'starred comments' to differentiate them from ordinary comments
```

Comment:
instead of calling them "starred" comments

Reply:
The syntax is actually /** */ (Notice the extra asterix at the start). This is actually what I meant here.